banks/urls.py

```diff
@@ -1,6 +1,19 @@
 from django.urls import path
+from rest_framework.authtoken.views import obtain_auth_token
+from .views import BankListApiView, BankViewSet, BankListGenericView
```

Comment:
preferred to have absolute imports
`from banks.view`

no action needed. just remember as a good practice